tests/bsim/bluetooth/host/l2cap/reassembly/peer/src/peer.c

```diff
@@ -0,0 +1,651 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
```

Comment:
Note you have a few 2023 in the copyrights.

Reply:
Thanks, will fix